### DISCUSSION TITLE
Is Teleport fully open source publicly? Even the Enterprise Edition?

### DISCUSSION THREAD

**Discussion by haneef95** [gravitational/teleport#44126] [General]
Hi,

e.g:
1. Bitwarden is a fully open-source project that is openly auditable.
2. Gitlab Enterprise Edition is of a similar situation with all the code openly available for the community/public to audit: https://gitlab.com/gitlab-org/gitlab/-/blob/master/ee/LICENSE?ref_type=heads

Thanks,

**Comment by webvictim** [comment]
Teleport Enterprise is not open source - it is licensed under a separate commercial license.

**Reply by haneef95** [reply]
Thanks. Just to confirm... Even the source code for Teleport Enterprise is not publicly available?
(Officially or by Community)

**Reply by webvictim** [reply]
Correct, Teleport Enterprise is not open source, so its source code is not available.

**Reply by haneef95** [reply]
I see. Thank you for the confirmation.
